src/hooks/use-fetch-wpcom-sites/types/sync-site.ts

```diff
@@ -0,0 +1,13 @@
+import type { SyncSupport } from './sync-support';
+
+export type SyncSite = {
```

Comment:
```suggestion
export type ConnectedSite = {
```
I think renaming this type would help clarify the code.

Reply:
It's used for both - conencted sites and transformed site of response from /sites of wordpress.com
So I would keep the name as is, to keep it not specific for connected sites. Otherwise it can confuse, as list of sites would have also type as ConnectedSite[]

Reply:
Hmm… "sync sites" just sounds quite vague, IMO. `ConnectableSite` maybe? `ConnectableSyncSite`? `RemoteSyncSite`?

Reply:
I see your point, but it should be something like `ConnectedOrRemoteSite`. It's the same for both cases. Proposed options by you sound specific for some 1 case. And I can't come up with some name which works good to tackle both cases: ConnectedSite and RemoteSite :(
I don't have strong preference, so if you think that some option looks best for you - I am ok to go with it. So just tell me WDYT about it :)

Reply:
@fredrikekelund I am thinking that we can call it just `Site`. I would like to have more unique name, to simplify `Command + F`, but I see only this as a good option.
Will be waiting for your opinion and decision.

Reply:
I think `Site` is even more likely to be confused with local Studio sites… Let's just leave this as is for now. `SyncSite` does imply a site in the sync feature of the app, which is understandable, I suppose. Some of the unclarities around this will likely be resolved with future refactorings anyway